.github/workflows/closed_issue_check.yml

```diff
@@ -0,0 +1,51 @@
+# Contains job to check that all the todos are resolved for the closed issue.
+
+name: Closed Issue Check
+
+on:
+  issues:
+    types: [closed, reopened]
+
+permissions:
+  issues: write
+
+jobs:
+  script_check:
+    name: Script Check
```

Comment:
Should be renamed to be specific to the check in question.

Reply:
Done.